sql/sql_parse.cc

```diff
@@ -3203,6 +3206,60 @@ int mysql_execute_command(THD *thd, bool first_level) {
       res = purge_master_logs_before_date(thd, purge_time);
       break;
     }
+    case SQLCOM_PURGE_UUID: {
+      if (check_global_access(thd, SUPER_ACL)) goto error;
+      rpl_sidno server_sidno = gtid_state->get_server_sidno();
+      rpl_sidno master_sidno = 0;
+      // We should not purge gtids received from current active master.
+      get_master_sidno(thd, master_sidno);
+      rpl_sidno_container sidnos;
+      char *token = strtok(lex->gtid_string, ",");
+      while (token) {
+        rpl_sid sid;
+        while (*token == ' ') token++;
```

Comment:
I think we may want the equivalent of SKIP_WHITESPACE here? \t and others were acceptable forms of whitespace to this command in 5.6.